src/App.tsx

```diff
@@ -20,14 +36,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => setQuery(event.target.value)}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={movie} />
```

Comment:
it is better to use naming that give more info than 'movie'

Reply:
agree, f.e. `movies`